tests/unit/test_class_cmds.py

```diff
@@ -525,6 +523,21 @@ class TST_MemberOfFamilyCollection {
                  '<METHOD NAME="DeleteNothing" PROPAGATED="false"'],
       'test': 'regex'},
      SIMPLE_MOCK_FILE, OK],
+
+    ['Verify class command enumerate with --association filter.',
+     ['enumerate', '--association', '--no'],
+     {'stdout': ['//FakedUrl/root/cimv2:TST_Lineage',
+                 '//FakedUrl/root/cimv2:TST_MemberOfFamilyCollection'],
+      'test': 'innows'},
+     SIMPLE_ASSOC_MOCK_FILE, OK],
+
+    ['Verify class command enumerate with --association filter.',
+     ['enumerate', '--no-association', '--no'],
+     {'stdout': ['//FakedUrl/root/cimv2:TST_FamilyCollection',
+                 '//FakedUrl/root/cimv2:TST_Person'],
+      'test': 'in'},
+     SIMPLE_ASSOC_MOCK_FILE, OK],
+
```

Comment:
Add test cases for the indication filters.

Reply:
DONE